tests/unit/data/test_byte_cursor.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2024 Google LLC
```

Comment:
Previously, the test files mirrored the structure of the source files. It might make sense to throw these into an execute_query directory?

Reply:
fixed.

Reply:
done.